plugins/shortcuts/src/Shortcuts.test.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { SidebarContext, ApiProvider, ApiRegistry } from '@backstage/core';
+import { wrapInTestApp, MockStorageApi } from '@backstage/test-utils';
+import { render, screen, waitFor } from '@testing-library/react';
+import { Shortcuts } from './Shortcuts';
+import { LocalStoredShortcuts, shortcutsApiRef } from './api';
+
+const apis = ApiRegistry.from([
+  [shortcutsApiRef, new LocalStoredShortcuts(MockStorageApi.create())],
+]);
+
+describe('Shortcuts', () => {
+  it('displays an add button', async () => {
+    render(
+      wrapInTestApp(
```

Comment:
for future reference, there's also an `await renderInTestApp` that encapsulates these two AND does the `act` thing in such a way that you can rely on not getting the pesky "side effect" warnings in the log